app/src/main/java/info/nightscout/androidaps/plugins/configBuilder/ConfigBuilderPlugin.kt

```diff
@@ -52,6 +57,14 @@ class ConfigBuilderPlugin @Inject constructor(
         rxBus.send(EventAppInitialized())
     }
 
+    override fun startGlunovoService(context: Context) {
```

Comment:
This should be done in `GlunovoPluginService::onStart`
... and stopped in `GlunovoPluginService::onStop`

Reply:
resolved